resources/dicts/events/lifegen_events/murder.json

```diff
@@ -362,12 +367,12 @@
         "You give an indignant huff as v_c once again tries to 'help' out with your duties. Who is the deputy here, you or v_c? Just because they are an elder does not mean they are suddenly imbued with the knowledge of StarClan! Fed up, you ask v_c to take a peaceful walk outside the camp. Hours later, when asked about the whereabouts of v_c, you simply shake your head and murmur that you have no idea."
     ],
     "murder leader kitten": [
-        "You encourage v_c to eat some shiny red berries you've found, saying that they're super tasty - better than freshkill even! Happily, they thank you for the snack. v_c is later found, lifeless, with deathberry juice dripping from their mouth.",
+        "You encourage v_c to eat some shiny red berries you've found, saying that they're super tasty - better than freshkill even! Happily, they thank you for the snack. v_c is later found, lifeless, having already gone to StarClan.",
```

Comment:
this can be changed to match the previous one

"You encourage v_c to eat some shiny red berries you've found, and after eating them, v_c is later found having travelled to meet Silverpelt early."